docs/guide/basic-usage.md

```diff
@@ -134,3 +134,19 @@ You can use it by adding it to the `$providers` property of the current command.
     ],
 ]
 ```
+
+
+Yii2 Advanced Template
+----------------------
+ 
+When you want to run faker in the Yii2 advanced template, you need to specify your template and fixtureData path as @tests doesn't exist. For example if you want to setup common fixtures, use the following config in console/config/main.php:
```

Comment:
By default Faker is setup to use:
  ```
  /**
     * @var string Alias to the template path, where all tables templates are stored.
     */
    public $templatePath = '@tests/unit/templates/fixtures';
    /**
     * @var string Alias to the fixture data path, where data files should be written.
     */
    public $fixtureDataPath = '@tests/unit/fixtures/data';
```

When trying to run it using starter documentation, it produces an error which does not hint directly what is wrong:

> Error: The template path "@tests/unit/templates/fixtures" does not exist

To help others trying to get their tests up-and-running I propose a section on how to set it up in an advanced template setup.